pkgs/by-name/ce/cen64/package.nix

```
@@ -11,15 +11,23 @@
 
 stdenv.mkDerivation rec {
   pname = "cen64";
-  version = "unstable-2022-10-02";
+  version = "unstable-2023-05-29";
```

```suggestion
  version = "0-unstable-2023-05-29";
```
per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning